CRITICAL - 2022-01-13 18:34:26 --> No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 18:57:44 --> Class 'Config\App\Filters\Auth' not found
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(376): CodeIgniter\Filters\Filters->run('/', 'before')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2022-01-13 18:59:25 --> Auth filter must have a matching alias defined.
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Filters\Filters.php(312): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('Auth')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(367): CodeIgniter\Filters\Filters->enableFilter('Auth', 'before')
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-01-13 19:01:14 --> Class 'Config\App\Filters\Cors' not found
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(376): CodeIgniter\Filters\Filters->run('/', 'before')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2022-01-13 19:07:13 --> syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting ')'
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(358): CodeIgniter\CodeIgniter->tryToRouteIt(NULL)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2022-01-13 19:07:20 --> Class 'Config\App\Filters\Auth' not found
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(376): CodeIgniter\Filters\Filters->run('/', 'before')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#4 {main}
CRITICAL - 2022-01-13 19:07:25 --> Auth filter must have a matching alias defined.
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Filters\Filters.php(312): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('Auth')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(367): CodeIgniter\Filters\Filters->enableFilter('Auth', 'before')
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-01-13 19:07:26 --> Auth filter must have a matching alias defined.
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Filters\Filters.php(312): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('Auth')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(367): CodeIgniter\Filters\Filters->enableFilter('Auth', 'before')
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-01-13 19:07:26 --> Auth filter must have a matching alias defined.
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Filters\Filters.php(312): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('Auth')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(367): CodeIgniter\Filters\Filters->enableFilter('Auth', 'before')
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-01-13 19:07:26 --> Auth filter must have a matching alias defined.
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Filters\Filters.php(312): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('Auth')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(367): CodeIgniter\Filters\Filters->enableFilter('Auth', 'before')
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-01-13 19:07:26 --> Auth filter must have a matching alias defined.
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Filters\Filters.php(312): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('Auth')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(367): CodeIgniter\Filters\Filters->enableFilter('Auth', 'before')
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-01-13 19:13:47 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:13:49 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:14:00 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:14:01 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:14:02 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:14:09 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:14:10 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:14:28 --> Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1819): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\fix\app\Models\Barang_model.php(12): CodeIgniter\Model->findAll()
#5 C:\xampp\htdocs\fix\app\Controllers\Barang.php(12): App\Models\Barang_model->getBarang()
#6 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(910): App\Controllers\Barang->index()
#7 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Barang))
#8 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-01-13 19:16:55 --> Class 'CodeIgniter\App\Filters\Cors' not found
#0 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(376): CodeIgniter\Filters\Filters->run('barang', 'before')
#1 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 C:\xampp\htdocs\fix\public\index.php(45): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\fix\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#4 {main}
